src/workspace/internal/workspace.h

```diff
@@ -38,6 +38,9 @@ class Workspace : public IWorkspace
     std::string name() const override;
     std::string title() const override;
 
+    std::vector<WorkspaceTag> tags() const override;
```

Comment:
It would be great to add an alias:

WorkspaceTagList = std::vector<WorkspaceTag>